lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -759,7 +760,6 @@ static uint32_t collectTypeInfo(Module *M, swift::Demangle::Demangler &Dem,
       // Bug-for-bug-compatibility. Not sure if this is correct.
       swift_flags |= eTypeIsPointer | eTypeHasValue;
       return swift_flags;
-      LLVM_FALLTHROUGH;
```

Comment:
Most of the `case`s in this switch use a fallthrough. Adrian do you know if it's correct for this one to be a return, not a fall through?